Installation instructions in the README omit an important detail

### What happened?

The README mentions as one of the prerequisites for installing the library: `📥 Add the public Koodivaramu registry to your project by following the instructions provided [here](https://koodivaramu.eesti.ee/e-gov/cvi/-/packages/385).` 

It's important to note however that the Koodivaramu page by default offers to copy a configuration setting for Instance-level setup, and that setting doesn't let a project to install the library – it causes a 404: 

```
npm i @egov/cvi-ng
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@egov/cvi-ng - Not found
npm ERR! 404
npm ERR! 404  '@egov/cvi-ng@*' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

So a library user should be informed that to install the library they need to pick the second radio button: 
![image](https://github.com/e-gov/cvi/assets/1833029/044c133e-e432-4a7c-8818-7ab1bb0619db)


### Reproduction steps

1. Create an Angular project
2. Follow the instructions given in the [README](https://github.com/e-gov/cvi/blob/main/README.md)
3. When configuring the @egov npm registry, copy the code given on [the Koodivaramu page](https://koodivaramu.eesti.ee/e-gov/cvi/-/packages/575) without any extra action
4. See the 404 error


### Screenshot

_No response_

### Package

Not applicable

### Version

any

### Reproduction URL

_No response_

### Browser

_No response_

### Browser version

_No response_

### Extra details about your environment

_No response_

### Relevant log output

```shell
npm i @egov/cvi-ng
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@egov/cvi-ng - Not found
npm ERR! 404
npm ERR! 404  '@egov/cvi-ng@*' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

--- a/README.md
+++ b/README.md
@@ -24,7 +24,7 @@ To discuss any issues, suggestions or questions, join our public [CVI Signal gro
 
 Follow these steps to install and integrate our library:
 
-1. 📥 Add the public Koodivaramu registry to your project by following the instructions provided [here](https://koodivaramu.eesti.ee/e-gov/cvi/-/packages/385).
+1. 📥 Add the public Koodivaramu registry to your project by following the instructions provided [here](https://koodivaramu.eesti.ee/e-gov/cvi/-/packages/385). In short, you will need to update your `.npmrc` file. Be sure to choose the second radio button in the Koodivaramu UI ("**project**-level registry setup").
 
 2. 📦 Install the necessary package to your project:
   - CSS framework: Use the command `npm i --save @egov/cvi-styles`
@@ -90,7 +90,7 @@ For example, to introduce a content selector that inserts custom content before 
 
 ## 🚀 Running the Storybook
 
-Run the following command to build documentation and start the Storybook locally: 
+Run the following command to build documentation and start the Storybook locally:
 
 `npm run storybook`
 
@@ -98,7 +98,7 @@ Run the following command to build documentation and start the Storybook locally
 
 To run Storybook locally using Docker, follow these steps:
 
-1. Build the Docker image with the following command: 
+1. Build the Docker image with the following command:
 
 `docker build -f ./libs/storybook/Dockerfile -t cvi-storybook .`
 
@@ -140,10 +140,10 @@ The project uses automatic screenshot testing via Chromatic.
 
 1. To run the tests, use the following command in your terminal: `npm run chromatic`
 2. Open the resulting URL and review the visual changes, accepting or denying them in the Chromatic UI.
-   
+
 💡 Chromatic CI also runs on every push. The action always passes (even when visual changes are detected) except for cases when a story is broken. Contributors and reviewers should check the results of the action (and accept or decline them in the Chromatic UI) by following a link in the build log. 💡
 
-Publishing to Chromatic also gives a possibility to share a Storybook link for a specific branch (even non-pushed, if the local npm command is used) in this format: 
+Publishing to Chromatic also gives a possibility to share a Storybook link for a specific branch (even non-pushed, if the local npm command is used) in this format:
 
 `https://<branch>--6373995e3f280e239470296d.chromatic.com`
 

